Repositories/BookingRepositoty.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AirportTicketBooking.Classes;
+
+namespace AirportTicketBooking.Controllers
+{
+    internal class BookingRepositoty
+    {
+        private const string filePath = "booking.csv";
+
+
+        internal static void SaveToFile(Booking booking)
+        {
+            var writer = new StreamWriter(filePath, true);
+
+            if (!File.Exists(filePath))
+            {
+                writer.WriteLine("BookingId,FlightId,PassengerPassportId," +
+                                   "BookingDate,ClassType");
+            }
+
+            string flightData = $"{booking.BookingId}, {booking.FlightId}, {booking.PassengerPassportId}," +
+                                 $"{booking.BookingDate}, {booking.ClassType}";
+
+            writer.WriteLine(flightData);
+        }
+
+        internal static List<object> LoadFromFile()
+        {
+            if (!File.Exists(filePath))
+            {
+                throw new Exception($"{filePath} does not exist.");
+            }
+
+            //make it object not Flight to add string when the reading line is Invaild format
+            var fileData = new List<object>();
+
+            var reader = new StreamReader(filePath);
+            string line;
+            reader.ReadLine(); //to skip the header line
+
+            while ((line = reader.ReadLine()) != null)
+            {
+                string[] bookingData = line.Split(',');
+
+                if (bookingData.Length != 5)
```

Comment:
The logic for reading the file and splitting the data is common for all repositories. This should inspire you to think in a way to generalize it and do this handling in one place and then just use it in all places.